Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -4739,6 +4739,9 @@ protected virtual bool InitSandboxedKextConnection(LoggingContext loggingContext
                             {
                                 ReportQueueSizeMB = m_configuration.Sandbox.KextReportQueueSizeMb,
                                 EnableReportBatching = m_configuration.Sandbox.KextEnableReportBatching,
+#if FEATURE_CORECLR

```

Comment:
Did you mean
```suggestion
#if PLATFORM_OSX
```
instead?

Reply:
Refactored, indeed.